applications/serial_lte_modem/doc/TWI_AT_commands.rst

```diff
@@ -0,0 +1,240 @@
+.. _SLM_AT_TWI:
+
+TWI AT commands
+***************
+
+.. contents::
+   :local:
+   :depth: 2
+
+The following commands list contains AT commands related to the two-wire interface (TWI).
+
+Open TWI instance #XTWILS
+=========================
+
+The ``#XTWILS`` command lists all available TWI instances.
+
+Set command
+-----------
+
+The set command allows you to list all available TWI instances.
+
+Response syntax
+~~~~~~~~~~~~~~~
+
+::
+
+   #XTWILS: <index>[[[,<index>],<index>],<index>]
+
+The ``<index>`` parameter corresponds to the following TWI instances:
+
+* ``0`` - TWI0 (``i2c0``).
+* ``1`` - TWI1 (``i2c1``).
+* ``2`` - TWI2 (``i2c2``).
+* ``3`` - TWI3 (``i2c3``).
+
+See :ref:`zephyr:dtbinding_nordic_nrf_twi`.
+
+Example
+~~~~~~~
+
+::
+
+   AT#XTWILS
+   #XTWILS: 2
+   OK
+
+Read command
+------------
+
+The read command is not supported.
+
+Test command
+------------
+
+The test command is not supported.
+
+Write to TWI peripheral device #XTWIW
+=====================================
+
+The ``#XTWIW`` command writes data to a TWI peripheral device.
+
+Set command
+-----------
+
+The set command allows you to write data to a TWI peripheral device.
+
+Syntax
+~~~~~~
+
+::
+
+   #XTWIW=<index>,<dev_addr>,<data>
+
+* The ``<index>`` parameter accepts the following integer values:
+
+  * ``0`` - Use TWI0 (``i2c0``).
+  * ``1`` - Use TWI1 (``i2c1``).
+  * ``2`` - Use TWI2 (``i2c2``).
+  * ``3`` - Use TWI3 (``i2c3``).
+
+See :ref:`zephyr:dtbinding_nordic_nrf_twi`.
```

Comment:
```suggestion
  See :ref:`zephyr:dtbinding_nordic_nrf_twi`.
```

Reply:
The formatting is messing up the list, so need to add indentation.

Reply:
modified.